packages/region/src/services/region-module.ts

```diff
@@ -117,78 +115,31 @@ export default class RegionModuleService<
     data: CreateRegionDTO[],
     @MedusaContext() sharedContext: Context = {}
   ): Promise<Region[]> {
-    let currencies = await this.currencyService_.list(
-      { code: data.map((d) => d.currency_code.toLowerCase()) },
-      {},
-      sharedContext
-    )
-
-    const countriesInDb = await this.countryService_.list(
-      { iso_2: data.map((d) => d.countries).flat() },
-      { select: ["iso_2", "region_id"] },
-      sharedContext
-    )
+    let normalizedInput = this.normalizeInput(data)
```

Comment:
nit: when a function doesnt rely on anything from the class context, I would say it could be a private static method on the class as an internal util.

Reply:
Yup agreed, will do it now.